browser/css/jsdialogs.css

```diff
@@ -521,6 +521,7 @@ input[type='checkbox']:checked.autofilter, .jsdialog input[type='checkbox']:chec
 	border-radius: var(--border-radius);
 	overflow: hidden;
 	align-self: center;
+	padding-left: 4px;
```

Comment:
Please avoid "left" "right" we also have to support Right-To-Left mode. We prefer "start" "end"
Please test with &lang=ar in the collabora online URL to see how it looks in arabic.

Reply:
@shehr7 https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values worth reading